cpp/tests/utilities/column_utilities.cu

```diff
@@ -295,27 +295,36 @@ std::vector<bitmask_type> bitmask_to_host(cudf::column_view const& c) {
 
 struct column_view_printer {
   template <typename Element, typename std::enable_if_t<is_numeric<Element>()>* = nullptr>
-  void operator()(cudf::column_view const& col, std::vector<std::string> & out) {
+  void operator()(cudf::column_view const& col, thrust::host_vector<std::string> & out) {
     auto h_data = cudf::test::to_host<Element>(col);
 
     out.resize(col.size());
 
     if (col.nullable()) {
-      std::transform(thrust::make_counting_iterator(size_type{0}),
-                     thrust::make_counting_iterator(col.size()),
-                     out.begin(),
-                     [&h_data](auto idx) {
-                       return bit_is_set(h_data.second.data(), idx) ? std::to_string(h_data.first[idx]) : std::string("NULL");
-                     });
+
+      std::transform(
+        thrust::make_counting_iterator(size_type{0}),
+        thrust::make_counting_iterator(col.size()),
+        out.begin(),
+        [&h_data](auto idx) {
+          return bit_is_set(h_data.second.data(), idx) ? std::to_string(h_data.first[idx]) : std::string("NULL");
+        });
+
     } else {
-      std::transform(h_data.first.begin(), h_data.first.end(), out.begin(), [](Element el) {
+
+      std::transform(
+        h_data.first.begin(), 
```

Comment:
Why do this, if you don't do it for all function calls? (Which I hope we wouldn't do, since it will unnecessarily make our code much longer to scroll through.)

Reply:
This just comes down to preference. I think it makes the code much more readable, arguably even beautiful - and there are many cases where this is they style being used. A really good example is in @karthikeyann recent Rank PR, in the file:

https://github.com/rapidsai/cudf/blob/branch-0.14/cpp/src/sort/rank.cu

Some of this code is beautiful/so much more readable because of the vertical formatting:

```cpp
template<typename outputType>
void rank_min(rmm::device_vector<size_type> const &group_keys,
              column_view sorted_order_view, mutable_column_view rank_mutable_view, cudaStream_t stream) {
  // min of first in the group
  // All equal values have min of ranks among them.
  // algorithm: reduce_by_key(dense_rank, 1, n, min), scatter
  tie_break_ranks_transform<size_type>(
      group_keys,
      thrust::make_counting_iterator<size_type>(1),
      sorted_order_view,
      rank_mutable_view.begin<outputType>(),
      thrust::minimum<size_type>{},
      thrust::identity<outputType>{},
      stream);
}

template<typename outputType>
void rank_max(rmm::device_vector<size_type> const &group_keys,
              column_view sorted_order_view, mutable_column_view rank_mutable_view, cudaStream_t stream) {
  // max of first in the group
  // All equal values have max of ranks among them.
  // algorithm: reduce_by_key(dense_rank, 1, n, max), scatter
  tie_break_ranks_transform<size_type>(
      group_keys,
      thrust::make_counting_iterator<size_type>(1),
      sorted_order_view,
      rank_mutable_view.begin<outputType>(),
      thrust::maximum<size_type>{},
      thrust::identity<outputType>{},
      stream);
}

// ...

    thrust::transform(
        rmm::exec_policy(stream)->on(stream),
        rank_iter,
        rank_iter + input.size(),
        rank_iter,
        [is_dense, drs, count] __device__(double r) -> double {
          return is_dense ? r / drs[count - 1] : r / count;
        });
```
As Walter Brown said in his Meeting C++ 2019 keynote: 
**Most people don't take advantage of their eyes**
https://youtu.be/OAmWHmwlMwI?t=3195

All this being said, for this case it it doesn't have as a big a trade-off when it comes to readability so I can switch it back